src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let result = days * 40;
+
+  if (days >= 7) {
+    result = result - 50;
+  } else if (days >= 3) {
+    result = result - 20;
```

Comment:
Replace the literals '3' and '20' with constants to avoid hardcoding. For example, 'const SHORT_TERM_RENTAL = 3;' and 'const SHORT_TERM_DISCOUNT = 20;'.